upload_analysis_files.py

```diff
@@ -73,6 +78,22 @@
         drive_client_wrapper.update_or_create(individuals_csv_input_path, individuals_csv_drive_dir,
                                               target_file_name=individuals_csv_drive_file_name,
                                               target_folder_is_shared_with_me=True, recursive=True)
+
+        log.info("Uploading Automated Analysis CSVs to Drive...")
```

Comment:
No need to address immediately, but since the directory structure of automated analysis is the same locally as on Drive I wonder if it would make sense to recursively upload all the files we find instead of uploading each known directory. That would mean we wouldn't need to edit this script between projects.

Reply:
I agree, will update in a future pr.